src/sass/_forms.scss

```
@@ -482,3 +501,53 @@
     margin-right: $sph--x-small;
   }
 }
+
+.network-select-dropdown {
+  div.label {
```

```suggestion
  .label {
```
When targeting classes, we usually avoid the specific element. This also applies to other css below.